### ISSUE TITLE
Unexpected token ERROR

### ISSUE THREAD

**Issue by dijs** [SublimeLinter/SublimeLinter#1516]
I have been trying to figure this out for months... 

I have uninstalled and reinstalled the package numerous times, tried changing settings, etc.

I am using Node v8.9 so the spread operator should be allowed.

I am only using a local version of eslint@5.8.0 installed in node_modules

```
SublimeLinter: #1 eslint Terrain.js ERROR:
==========================================

.../node_modules/eslint/lib/linter.js:288
                disableDirectives.push(...createDisableDirectives(directiveType, comment.loc.start, directiveValue));
                                       ^^^

SyntaxError: Unexpected token ...
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (.../node_modules/eslint/lib/cli-engine.js:21:14)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
```

**Comment by dijs** [comment]
Oh wow. I just had the idea of downgrading eslint to "4.19.1"

That actually worked!

Soo.... SublimeLinter doesn't like something in v5 of eslint

**Comment by rchl** [comment]
If it throws syntax error on spread operator then it must be using older version of node. Maybe run this in Sublime's console and see what it outputs:

```
import subprocess
print(subprocess.check_output(['node', '--version']))
```

**Comment by kaste** [comment]
The typical `npm` 'binary' (found in 'node_modules/.bin') just executes `node ./node_modules/eslint/bin/eslint.js` so it really depends on `which node` at that exact time. (If 'node_modules/.bin/node' exists it uses that btw.)

**Comment by rishisharma08** [comment]
i am getting this problem as well.
whats the info that is needed, maybe i can provide?
